apps/emqx_authn/src/emqx_authn_api.erl

```diff
@@ -781,16 +781,16 @@ fill_defaults(Configs) when is_list(Configs) ->
 fill_defaults(Config) ->
     emqx_authn:check_config(Config, #{only_fill_defaults => true}).
 
-convert_certs(#{<<"ssl">> := SSLOpts} = Config) ->
+convert_certs(#{ssl := SSLOpts} = Config) ->
```

Comment:
doesn't this function work on the raw config?

Reply:
for the record: it is indeed raw config as input, but filled with defaults, and checked with `atom_keys` option